app/assets/javascripts/app/app.js

```diff
@@ -170,5 +170,6 @@ var app = {
 
 $(function() {
   app.initialize();
+  $('form').areYouSure();
```

Comment:
I'm not sure we should do this for every single form, especially search boxes come to mind where it's unwanted.

Also if we're adding a plugin for this, we probably should replace the existing custom code for the publisher with it.